modules/ROOT/pages/usage-reports.adoc

```diff
@@ -38,9 +38,9 @@ To calculate usage, MuleSoft tracks the number of managed API instances each bus
 
 Runtime reports track your use of Mule messages, Mule flows, and data throughput. 
 
-To track Mule message usage, the report counts the number of times a Mule event source triggers a Mule message. You can view the number of these messages in a given month by business group, environment, and application. To calculate usage, MuleSoft meters and aggregates the number of messages on a monthly basis. After a message is triggered, the report does not track changes to the message because it is processed within the application’s flows.
+To track Mule message usage, the report counts the number of times a Mule event source triggers a Mule message. You can view the number of these messages in a given month by business group, environment, and application. To calculate usage, MuleSoft meters and aggregates the number of messages on a monthly basis. After a message is triggered, the report does not track changes to the message as it is processed within the application’s flows.
```

Comment:
```suggestion
To track Mule message usage, the report counts the number of times a Mule event source triggers a Mule message. You can view the number of these messages in a given month by business group, environment, and application. To calculate usage, MuleSoft meters and aggregates the number of messages on a monthly basis. After a message is triggered, the report does not track changes to the message because the message is processed within the application’s flows.
```